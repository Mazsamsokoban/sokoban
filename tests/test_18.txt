loadtest 18
showstate shole
showstate sw
move w right
showmaze
showstate shole
showstate sw