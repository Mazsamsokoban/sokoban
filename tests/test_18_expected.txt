shole:null f1:w f2:box sw:null 

name:shole
friction:Normal
thing:null
open:false

name:sw
friction:Normal
thing:null
on:false
switchablehole:shole
box:null

shole:null f1:null f2:w sw:box 

name:shole
friction:Normal
thing:null
open:true

name:sw
friction:Normal
thing:box
on:true
switchablehole:shole
box:box

